specs/differences-from-optimism-bedrock.md

```diff
@@ -23,33 +23,57 @@
-| `validator` | `L2 EL client` + `L2 CL client` + `kroma-validator` + (optionally `kroma-prover`)   |
-| `vanilla`   | `L2 EL client` + `L2 CL client`                                                     |
+It is crucial to have at least one honest verifier who can verify the integrity of the rollup chain to ensure the
+ongoing security of the network. However, there exists a well-known challenge known as the 'Verifier's Dilemma' that
+poses a threat to the security of optimistic rollups by introducing disincentives in such scenarios.
 
-`kroma-prover` is only needed when `kroma-validator` is turned on with `challenger` mode.
-
-**NOTE:** Here `L2 EL client` means `kroma-geth` and `L2 CL client` means `kroma-node`. `L2 EL client` can
-be expanded to other clients for pragmatic decentralization.
+To resolve the 'Verifier's Dilemma', we have devised an incentive mechanism that motivates node operators to actively
+participate in the Kroma network. As part of this redesign, we have separated the responsibility of submitting
+checkpoint outputs from `sequencers` and assigned it to `verifiers`. As a result, these participants have been renamed
+to reflect these role changes and the future direction of Kroma's decentralization. For more detailed information about
+our decentralization scheme, please refer to
+[this article](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)
+on the Kroma blog.
 
 ### Sequencer -> Proposer
 
-We use `sequencing` for transaction inclusion, exclusion and ordering. Currently, `sequencer` does not only
-this but also does block building. Thinking of Ethereum PoS, this is what `proposer` exactly does, which is building
-and proposing a block. To take this into consideration, we rename the actor `sequencer` to `proposer` and plan to
-separate block building ability from `proposer` in the future. We will make the results of our research public.
+We use *sequencing* for the inclusion, exclusion and ordering of transactions. Currently, the `sequencer`
+not only performs these functions but also proposes built blocks to Layer 1. In Ethereum PoS, the `proposer` is
+responsible for precisely what the `sequencer` does - building and proposing a block. With this in mind, we have decided
+to rename the actor from `sequencer` to `proposer` and intend to separate the block building capability from the
+proposer in the future. We are comitted to sharing the results of our research with the public.
+
+### Verifier -> Validator
+
+We utilize the term `validator` to denote a participant who is responsible for submitting the
+[L2 output root][g-l2-output-root] and validating its accuracy by either submitting dispute challenges (during the
+optimistic rollup phase) or providing zk validity proofs (during the zk rollup phase). This concept bears a resemblance
+to how L1 validators cast FFG votes at each epoch.
 
-### Proposer -> Validator
+### Compositions
```

Comment:
```suggestion
### Composition
```

Reply:
both 'composition' and 'compositions' are valid for the plural form of 'composition'.
I'd like to use 'compositions' to indicate there are various configurations to compose different types of nodes.